.gitignore

```diff
@@ -1,2 +1,3 @@
 .idea/
 node_modules/
+coverage/
```

Comment:
If you add a rule for ignoring, you should delete the folders/files manually, as it doesn't apply to the previous commits.